lib/src/fty_mdns_sd_server.cc

```diff
@@ -306,37 +330,31 @@ s_handle_pipe(fty_mdns_sd_server_t* self, zmsg_t **message_p)
         zstr_free(&stype);
         zstr_free(&port);
     }
-    else
-    if (streq (command, "SET-DEFAULT-TXT")) {
+    else if (streq (command, "SET-DEFAULT-TXT")) {
         char *key   = zmsg_popstr (message);
         char *value = zmsg_popstr (message);
         log_debug("fty-mdns-sd-server: SET-DEFAULT-TXT %s=%s",
             key,value) ;
-         s_set_txt_record(self,key,value);
+        s_set_txt_record(self,key,value);
         zstr_free(&key);
         zstr_free(&value);
     }
-    else
-    if (streq (command, "DO-DEFAULT-ANNOUNCE")) {
+    else if (streq (command, "DO-DEFAULT-ANNOUNCE")) {
         //free previous value
         zstr_free (&self->fty_info_command);
         //get info from fty-info
         self->fty_info_command = zmsg_popstr (message);
-        log_debug("fty-mdns-sd-server: DO-DEFAULT-ANNOUNCE %s",
-                self->fty_info_command);
+        log_debug("fty-mdns-sd-server: DO-DEFAULT-ANNOUNCE %s", self->fty_info_command);
         int rv = -1;
         int tries = 3;
         while (tries-- >= 0) {
-            rv=s_poll_fty_info(self);
+            rv = s_poll_fty_info(self);
             if (rv == 0)
                 break;
-            // Wait 5 seconds before retrying
-            if (tries == 0)
-                zclock_sleep (5000);
         }
         // sanity check, this should trigger a service abort then restart
         // in the worst case, if we did not succeeded after 3 tries
-        assert(rv==0);
+        assert(rv == 0);
```

Comment:
I'm still not a fan of these Zstyle assert to sigabort!

Reply:
totally agree. we got once gigatons of coredump due to that assert... quick code, bad code?